_rules/__tests__/hyperlinks.js

```diff
@@ -0,0 +1,26 @@
+const describeRule = require('../../test-utils/describe-rule')
+const markdownLinkExtractor = require('markdown-link-extractor')
+const isUrl = require('is-url')
+
+const blacklistOfLinks = [
```

Comment:
I think it would be nice to add some comment as to what is the correct URL, it's not a one-to-one relation, but something like:
```javascript
'://www.w3.org/TR/WCAG20/', // Use WCAG 2.1
…
'://www.w3.org/TR/html/' // use spec.whatwg.org
```
would be nice (and easy to do). 

Maybe even go all the way and have the comment/message be printed out in the test in case of error, so that people looking at test will immediately get the hint as to what to do to fix it (but that's more work…)